test/API/CMakeLists.txt

```diff
@@ -0,0 +1,314 @@
+# Copyright by The HDF Group.
```

Comment:
It may be debatable, but for consistency we include the following lines at the top of CMakeLists.txt:
cmake_minimum_required (VERSION 3.18)
project (HDF5_XXXX C)